### DISCUSSION TITLE
How to change the repo's visibility?

### DISCUSSION THREAD

**Discussion by lunarforky-z** [community/community#143723] [Repositories]
### Select Topic Area

Question

### Body

I want to know how I can change the github repo's visibility.

**Comment by hylcore-V** [comment]
u can find the setting button in the top of the repository.
and click the button and scroll down to the bottom.
and now u can see the change visibility button.
after clicking the button,  the repo's visibility will be changed

**Comment by lunarforky-z** [comment]
Thank you for your advice
